client/cl_main.lua

```diff
@@ -90,14 +103,14 @@ local function PlacingThread(animData)
                     heading = heading - 5
                     if heading < 0 then heading = 360.0 end
                 end
-                if IsControlJustPressed(0, 38) then
-                    if #(GetEntityCoords(PlayerPedId()) - currentCoords) < 5.0 then
+                if IsControlJustReleased(0, 38) then
+                    if #(GetEntityCoords(PlayerPedId()) - currentCoords) < 35.0 then
```

Comment:
this is just exploitable. This means a player can push the ped 35 units away and still just walk all the way there.

Reply:
my bad, i forgot to revert these. testing